assets/css/styles.scss

```diff
@@ -26,6 +27,34 @@ h1, h2, h3, h4, h5, h6 {
   vertical-align: middle;
 }
 
+.button-reset {
```

Comment:
why was it added? was it specifically needed?

Reply:
I use it to reset some default button styles for the restaurant menu items. Now I think that it does not need to be a global style. Moved it to the menu.scss